### DISCUSSION TITLE
Selection of multiple information risk acceptance thresholds

### DISCUSSION THREAD

**Discussion by K8Sudo** [monarc-project/MonarcAppFO#372] [Ideas]
In the current display of risks, a representation in the three colors red, orange and yellow is possible. Here it would be helpful to allow a free selection of the threshold values. In many larger organizations, there are already methods in place that require reporting at different levels on the basis of several threshold values. Currently, this cannot be mapped so nicely.

A free color selection of the risk levels would also be helpful.

Greetings

Marc

![grafik](https://user-images.githubusercontent.com/47662401/131818048-f07bb5bd-2594-450d-a189-74a41de8c1b6.png)

**Comment by ruslanbaidan** [comment]
During Monarc workshops we had similar requests and this feature can be considered as a future tool improvement.
Thank you.
